src/vivarium/framework/resource.py

```diff
@@ -144,24 +149,24 @@ def sorted_nodes(self):
         """
         if self._sorted_nodes is None:
             try:
-                self._sorted_nodes = list(nx.algorithms.topological_sort(self.graph))
+                self._sorted_nodes = list(nx.algorithms.topological_sort(self.graph))  # type: ignore[func-returns-value]
```

Comment:
This was giving an error that `nx.algorithms.topological_sort` alwasy returns None which simply isn't true. I didn't dig too deep but assume that the stub package I downloaded (which is not by the same folks who maintain networkx) simply typed it wrong.

Reply:
Is it worth contributing a PR to the stubs package to fix this?

Reply:
Interesting, I'll take a look. I've never contributed to a 3rd party package before so that would be fun